docs/management_api.yml

```diff
@@ -183,6 +183,76 @@ paths:
         500:
           $ref: "#/responses/InternalServerError"
 
+
+  /deployments/statistics/list:
+    post:
+      operationId: Deployment Status Statistics
+      tags:
+        - Management API
+      security:
+        - ManagementJWT: []
+      summary: |
+        Get status count for all devices in the listed deployments (plural).
+      parameters:
+        - name: deployment_ids
+          in: body
+          description: Deployment identifier
+          required: true
+          schema:
+            type: array
+            items:
+                type: string
+                description: >-
+                    The list of deployment IDs
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          examples:
+            application/json:
+              - id: 0c13a0e6-6b63-475d-8260-ee42a590e8ff
+                statistics:
```

Comment:
```suggestion
                stats:
```
I'm so sorry!

Reply:
No, my fault. The code adheres to `stats`, so good find!
:)

Reply:
https://github.com/mendersoftware/deployments/pull/666/commits/cf41a79aefa61eee8e49d655d0664f038b0becb1